dynamic_forms/widgets.py

```diff
@@ -44,3 +44,6 @@ def id_for_label(self, id):
 
     def get_context(self):
         return {'name': ''}
+    
+    def value_from_datadict(self, data, files, name):
+        return data.get(name)
```

Comment:
Why is this change necessary?

Reply:
This was required as it was missing that method to create this fake model and Django was generating an error.